spec/wizards/placements/add_placement/steps/mentors_spec.rb

```diff
@@ -0,0 +1,76 @@
+require "rails_helper"
+
+RSpec.describe Placements::AddPlacement::Steps::Mentors, type: :model do
+  describe "attributes" do
+    it { is_expected.to have_attributes(school: nil, mentor_ids: []) }
+  end
+
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:school) }
+    it { is_expected.to validate_presence_of(:mentor_ids) }
+  end
+
+  describe "#mentors_for_selection" do
+    it "returns mentors for the school" do
+      school = create(:placements_school)
+      mentor = create(:placements_mentor_membership, school:).mentor
+      step = described_class.new(school:)
+
+      expect(step.mentors_for_selection).to eq([mentor.becomes(Placements::Mentor)])
+    end
+  end
+
+  describe "#mentor_ids=" do
+    context "when the value is blank" do
+      it "remains blank" do
+        school = create(:placements_school)
+        step = described_class.new(school:)
+
+        step.mentor_ids = []
+
+        expect(step.mentor_ids).to eq([])
+      end
+    end
+
+    context "when the value includes 'not_known'" do
+      it "removes all values except not_known" do
+        school = create(:placements_school)
+        mentor = create(:placements_mentor_membership, school:).mentor
+        step = described_class.new(school:)
+
+        step.mentor_ids = ["not_known", mentor.id]
+
+        expect(step.mentor_ids).to eq([:not_known])
+      end
+    end
+
+    context "when the value includes mentor ids" do
+      it "retains the mentor ids" do
+        school = create(:placements_school)
+        mentor = create(:placements_mentor_membership, school:).mentor
+        step = described_class.new(school:)
+
+        step.mentor_ids = [mentor.id]
+
+        expect(step.mentor_ids).to eq([mentor.id])
+      end
+    end
+  end
```

Comment:
Do we also want a scenario to cover when the array contains random strings and/or mentor IDs which belong to _a different school_?

The point being, those will be filtered out because we do:

```ruby
school.mentors.where(id: mentor_ids).ids
```

This seems like a valuable behaviour to cover with a test.